.eslintrc.js

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
I did a compare and can confirm nothing has changed in this file apart from changing from JSON to JS